### PR TITLE
Add health worker registrations to report sheet

### PR DESCRIPTION
**Pull request by rudigiesler**
We need to add a second sheet to the report export, that has registrations per health worker.

### CODE REVIEW

**Comment by rudigiesler** [comment]
Ready for review

File: ci/management/commands/generate_reports.py [hunk 1]
```diff
@@ -267,3 +271,35 @@ def handle_registrations(self, sheet, hub_client, ids_client,
...
+            created_after=start_date.isoformat(),
+            created_before=end_date.isoformat())
+
+        registrations_per_operator = collections.defaultdict(int)
+
+        for registration in registrations:
+            operator_id = registration.get('data', {}).get('operator_id')
+            registrations_per_operator[operator_id] += 1
+
+        for operator_id, count in registrations_per_operator.iteritems():
```
Comment (erikh360) [inline ci/management/commands/generate_reports.py]: Replace this with `.items()` for python v2 and v3 compatibility.

File: ci/tests.py [hunk 2]
```diff
@@ -237,6 +237,155 @@ def test_generate_report(self):
...
+    def test_generate_report_health_worker_registrations(self):
+        """
+        When generating a report, the second tab should be registrations per
+        health worker, and it should have the correct information.
+        """
+
+        # Registrations, first page, just returns empty results to make sure
+        # we're actually paging through the results sets using the `next`
+        # parameter
+        responses.add(
```
Comment (erikh360) [inline ci/tests.py]: We should maybe move adding the responses out into reusable functions if we are going to create a test function for each report tab

**Review by erikh360 — COMMENTED** [review]
Looks good, just the 2 comments.

**Review by erikh360 — APPROVED** [review]

**Pull request merged**